### PR TITLE
Some small updates to FH scenario rules

### PR DESCRIPTION
**Pull request by TheSquinter**
Some tidying up of special rules and boss abilities

### CODE REVIEW

**Review by Lurkars — CHANGES_REQUESTED** [review]
Have added some questions...

File: data/fh/scenarios/103.json [hunk 1]
```diff
@@ -28,54 +28,7 @@
...
-        {
-          "monster": {
-            "name": "lurker-clawcrusher",
-            "type": "normal"
-          },
-          "count": "0",
-          "manual": true,
-          "manualMax": 4
-        }
-      ]
```
[the remark this answers is not in the document]
Comment (TheSquinter) [reply data/fh/scenarios/103.json:78]: I thought it looked too congested to happen each round, especially on mobile and when not all monsters would be needed.
image

This now reminds them and players can manually spawn the monsters as needed.

File: data/fh/sections/97-3.json [hunk 2]
```diff
@@ -51,12 +51,6 @@
         }
       ]
     },
-    {
-      "round": "true",
-      "always": true,
-      "once": true,
-      "note": "All characters and character summons gain disadvantage on all their attacks"
-    },
```
[the remark this answers is not in the document]
Comment (TheSquinter) [reply data/fh/sections/97-3.json:59]: It's already added automatically via permanent condition muddle to all characters from the previous section.

File: data/fh/sections/167-1.json [hunk 3]
```diff
@@ -5,18 +5,6 @@
...
-      "name": "Ice Pillar",
-      "health": "[(L+1)xC/2{$math.floor}]",
-      "count": 6
-    },
-    {
-      "name": "Large Cave Rock",
-      "health": "[(L+1)xC/2{$math.floor}]",
-      "count": 3
-    }
-  ],
```
[the remark this answers is not in the document]
Comment (TheSquinter) [reply data/fh/sections/167-1.json:19]: The objectives were already defined in parent 54 and have the tags added there for the `rules` of opening sections when X objectives are killed.

**Pull request merged**